docs/reference/commandline/events.md

```diff
@@ -30,7 +30,7 @@ Options:
 
 Docker containers report the following events:
 
-    attach, commit, copy, create, destroy, detach, die, exec_create, exec_detach, exec_start, export, health_status, kill, oom, pause, rename, resize, restart, start, stop, top, unpause, update
+    archive-path, attach, checkpoint, commit, copy, create, destroy, detach, die, exec_create, exec_detach, exec_start, export, extract-to-dir, health_status, kill, oom, pause, rename, resize, restart, start, stop, top, unpause, update
```

Comment:
Thanks for doing this @allencloud; the `checkpoint` events should probably not be documented for now, because they're still experimental (I don't think we document experimental events, because they can still change so are not really part of the "specs")


I'm having a discussion right now with @vdemeester about the `archive-path` and `extract-to-dir` events. They appear to be implemented as part of https://github.com/docker/docker/pull/13171 (https://github.com/docker/docker/commit/c32dde5baadc8c472666ef9d5cead13ab6de28ea), but I think they're really "odd".

- They don't follow the same naming conventions as other events (dash instead of underscore)
- It's unclear what their purpose is (they're related to `copy` - if they're important, they should probably be renamed to something like `copy_start`, `copy_complete`)

I'm not sure what the intended purpose was for these events; if it's to collect metrics, then maybe they should be moved to the Prometheus API.

Let's not document these for now, and not "bake" them into the API, I think these need discussion

ping @jlhawn ^^